060_Distributed_Search/20_Scan_and_scroll.asciidoc

```diff
@@ -75,7 +65,3 @@ _previous_ scroll request.
 
 When no more hits are returned, we have processed all matching documents.
 
-TIP: Some of the http://www.elastic.co/guide[official Elasticsearch clients]
```

Comment:
Curiosity question: why remove this?  Perhaps better to link directly to the perl/python helpers that do this?